### PR TITLE
Revert Coretiming PRs 8531 and 7454

### PR DESCRIPTION
**Pull request by Kelebek1**
Requested by Pharynx to fix some low FPS and freezing issues for some users.

### CODE REVIEW

File: src/common/thread.cpp [hunk 1]
```diff
@@ -47,9 +47,6 @@ void SetCurrentThreadPriority(ThreadPriority new_priority) {
     case ThreadPriority::VeryHigh:
         windows_priority = THREAD_PRIORITY_HIGHEST;
         break;
-    case ThreadPriority::Critical:
-        windows_priority = THREAD_PRIORITY_TIME_CRITICAL;
-        break;
     default:
         windows_priority = THREAD_PRIORITY_NORMAL;
         break;
```
Comment (FernandoS27) [inline src/common/thread.cpp:52]: don't remove this.

File: src/common/thread.cpp [hunk 2]
```diff
@@ -62,10 +59,9 @@ void SetCurrentThreadPriority(ThreadPriority new_priority) {
 void SetCurrentThreadPriority(ThreadPriority new_priority) {
     pthread_t this_thread = pthread_self();
 
-    const auto scheduling_type = SCHED_OTHER;
-    s32 max_prio = sched_get_priority_max(scheduling_type);
-    s32 min_prio = sched_get_priority_min(scheduling_type);
-    u32 level = std::max(static_cast<u32>(new_priority) + 1, 4U);
```
Comment (FernandoS27) [inline src/common/thread.cpp:68]: ditto.

File: src/common/thread.h [hunk 3]
```diff
@@ -92,7 +92,6 @@ enum class ThreadPriority : u32 {
     Normal = 1,
     High = 2,
     VeryHigh = 3,
-    Critical = 4,
```
Comment (FernandoS27) [inline src/common/thread.h:95]: ditto.

File: src/common/x64/native_clock.cpp [hunk 4]
```diff
@@ -89,7 +89,8 @@ u64 NativeClock::GetRTSC() {
         new_time_point.inner.accumulated_ticks = current_time_point.inner.accumulated_ticks + diff;
     } while (!Common::AtomicCompareAndSwap(time_point.pack.data(), new_time_point.pack,
                                            current_time_point.pack, current_time_point.pack));
-    return new_time_point.inner.accumulated_ticks;
+    /// The clock cannot be more precise than the guest timer, remove the lower bits
+    return new_time_point.inner.accumulated_ticks & inaccuracy_mask;
```
Comment (FernandoS27) [inline src/common/x64/native_clock.cpp:93]: don't revert this.

File: src/common/x64/native_clock.h [hunk 5]
```diff
@@ -37,6 +37,11 @@ class NativeClock final : public WallClock {
         } inner;
     };
 
+    /// value used to reduce the native clocks accuracy as some apss rely on
+    /// undefined behavior where the level of accuracy in the clock shouldn't
+    /// be higher.
+    static constexpr u64 inaccuracy_mask = ~(UINT64_C(0x400) - 1);
+
```
Comment (FernandoS27) [inline src/common/x64/native_clock.h:44]: erase this.

File: src/core/core_timing.cpp [hunk 6]
```diff
@@ -44,11 +42,11 @@ CoreTiming::CoreTiming()
...
-void CoreTiming::ThreadEntry(CoreTiming& instance, size_t id) {
-    const std::string name = "yuzu:HostTiming_" + std::to_string(id);
-    MicroProfileOnThreadCreate(name.c_str());
-    Common::SetCurrentThreadName(name.c_str());
-    Common::SetCurrentThreadPriority(Common::ThreadPriority::Critical);
+void CoreTiming::ThreadEntry(CoreTiming& instance) {
+    constexpr char name[] = "yuzu:HostTiming";
+    MicroProfileOnThreadCreate(name);
+    Common::SetCurrentThreadName(name);
+    Common::SetCurrentThreadPriority(Common::ThreadPriority::VeryHigh);
```
Comment (FernandoS27) [inline src/core/core_timing.cpp:49]: keep it as Critical

**Comment by german77** [comment]
Breaks audio

**Review by liamwhite — APPROVED** [review]

**Pull request merged**